packages/front-end/pages/settings/webhooks.tsx

```diff
@@ -26,8 +26,13 @@ const WebhooksPage: FC = () => {
           </a>
         </Link>
       </div>
-      <h1>Webhooks</h1>
+      <h1>SDK Endpoint Webhooks</h1>
       <Webhooks />
+
+      {/* TODO: Include this code when we are ready to launch Event-based Web Hooks */}
+      {/*<div className="mt-5">*/}
+      {/*  <EventWebHooksPage />*/}
+      {/*</div>*/}
```

Comment:
If you'd like to test the create flow, you'll need to uncomment this code so it can load under the existing web hooks.